src/asm_ghidra.cpp

```diff
@@ -5,33 +5,41 @@
 #include "SleighAsm.h"
 
 static SleighAsm sasm;
+static RIO *rio = nullptr;
```

Comment:
We will have to get rid of these globals in the long run and move them into some plugin-local state. But for now it's ok.